README.md

```diff
@@ -367,15 +367,20 @@ CREATE OR REPLACE FUNCTION ...;
 DROP FUNCTION IF EXISTS ... CASCADE;
 CREATE OR REPLACE FUNCTION ...
 ```
+## Disable Transaction
+Some migrations require execution outside of a transaction (e.g. to enable augmenting non-DDL-safe things, such as ENUMs in PostgreSQL). To disable wrapping a given migration file in a transaction, use the special comment `--! no-transaction` at the top of the migration file, e.g.
+
+```sql
+--! no-transaction
+ALTER TYPE user_role ADD VALUE 'Admin';
```

Comment:
Because we encourage idempotency, we should make this command idempotent:

```suggestion
ALTER TYPE user_role ADD VALUE IF NOT EXISTS 'Admin';
```

Reply:
Yes, it works.

Reply:
Although I should note, it only works for one statement. After that, it seems to wrap the rest in a  transaction.

Reply:
Really? I'm not sure that's what's happening.